glusterd2/commands/volumes/volume-stop.go

```diff
@@ -118,35 +119,44 @@ func registerVolStopStepFuncs() {
 func volumeStopHandler(w http.ResponseWriter, r *http.Request) {
 
 	ctx := r.Context()
-	ctx, span := trace.StartSpan(ctx, "/volumeStopHandler")
```

Comment:
as this is the main function with HTTP handler i think tracer should start from here only

Reply:
What if VoumeStop is called directly? Volume Create and Start had done it this way, hence followed the same